plugins/fs/guest-js/index.ts

```diff
@@ -1093,42 +1093,76 @@ interface WatchOptions {
 interface DebouncedWatchOptions extends WatchOptions {
   /** Debounce delay */
   delayMs?: number;
+  /** Keep track of the file system IDs of all files */
+  trackFileIds?: boolean;
```

Comment:
can we keep this in another PR, so this PR would be about migration only and also allow for a better discussion of the option